src/Controller/StoreApiController.php

```diff
@@ -185,6 +209,63 @@ public function getPaymentMethods(SalesChannelContext $context): JsonResponse
         return new JsonResponse($this->paymentMethodsService->getPaymentMethods($context));
     }
 
+    /**
+     * @Route(
+     *     "/store-api/adyen/payment-methods/balance",
+     *     name="store-api.action.adyen.payment-methods.balance",
+     *     methods={"POST"}
+     * )
+     *
+     * @param SalesChannelContext $context
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function getPaymentMethodsBalance(SalesChannelContext $context, Request $request): JsonResponse
+    {
+        $number = $request->request->get('number');
+        $type = $request->request->get('type');
+        $cvc = $request->request->get('cvc');
+
+        return new JsonResponse(
+            $this->paymentMethodsBalanceService->getPaymentMethodsBalance($context, $type, $number, $cvc)
+        );
+    }
+
+    /**
+     * @Route(
+     *     "/store-api/adyen/orders",
+     *     name="store-api.action.adyen.orders",
+     *     methods={"POST"}
+     * )
+     *
+     * @param SalesChannelContext $context
+     * @return JsonResponse
+     */
+    public function getOrders(SalesChannelContext $context, Request $request): JsonResponse
+    {
+        $orderId = $request->request->get('orderId');
```

Comment:
At this point we won't yet have a shopware order ID, this should actually be a create orders endpoint